### PR TITLE
Remove all feature flags

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
We don't have any active feature flags

### Changes proposed in this pull request
Remove all config

### Guidance to review
N/A

### CODE REVIEW

File: src/ui/Startup.cs [hunk 1]
```diff
@@ -236,8 +236,6 @@ public void ConfigureServices(IServiceCollection services)
                     }
                 };
             });
-            services.AddScoped<SearchAndCompare.UI.Shared.Features.IFeatureFlags, SearchAndCompare.UI.Shared.Features.FeatureFlags>();
```
Comment (timabell) [inline src/ui/Startup.cs]: This one is still in use

**Comment by timabell** [comment]
[remove old live-view feature flag from manage-ui](https://trello.com/c/DCmEm18k/502-remove-old-live-view-feature-flag-from-manage-ui)

File: src/ui/Startup.cs (hunk 1 above)
Comment (dankmitchell) [reply src/ui/Startup.cs]: @timabell I don't see how it can be... `IFeatureFlags` has been deleted 🤔

Comment (timabell) [reply src/ui/Startup.cs]: ah, but this is the *other* `IFeatureFlags` from the nuget package from search. The preview uses it

Comment (defong) [reply src/ui/Startup.cs]: Don't delete that, bad things will happen

**Pull request merged**